test-case/check-smart-amplifier.sh

```diff
@@ -98,14 +98,16 @@ do
         file="$tmp_dir/tmp_wave_${fmt%_*}.wav"
         recorded_file="$tmp_dir/recorded_tmp_wave_${fmt%_*}.wav"
         wavetool.py -gsine -A0.8 -B"${fmt%_*}" -o"$file"
-        dlogc "aplay -D$pb_dev -r $pb_rate -c $pb_chan -f $fmt -d $duration -q $file &"
-        aplay -D"$pb_dev" -r "$pb_rate" -c "$pb_chan" -f "$fmt" -d "$duration" -v -q "$file" &
-        dlogc "arecord -D$cp_dev -r $cp_rate -c $cp_chan -f $fmt -d $duration -q $recorded_file"
-        arecord -D"$cp_dev" -r "$cp_rate" -c "$cp_chan" -f "$fmt" -d "$duration" -v -q "$recorded_file"
+        dlogc "arecord -D$cp_dev -r $cp_rate -c $cp_chan -f $fmt -d $duration -v -q $recorded_file &"
+        arecord -D"$cp_dev" -r "$cp_rate" -c "$cp_chan" -f "$fmt" -d "$duration" -v -q "$recorded_file" &
+        sleep 0.5
+        dlogc "aplay -D$pb_dev -r $pb_rate -c $pb_chan -f $fmt -d $duration -v -q $file"
```

Comment:
@aiChaoSONG any idea why the 0.5 sleep? What happens without it?

Reply:
There is issue with DSM feature on master firmware, without the delay, we will get IPC timeout, and the firmware will not response anymore, unless reboot. In fact, 0.5 is an experiment value, 1 second should also work.

Reply:
Hm so similar issue with us except in our case it's just regular playback, two aplay calls in a row. Any half-idea on what could cause that?

Reply:
@paulstelian97  add comment here may get more response, https://github.com/thesofproject/sof/issues/3249

Reply:
Hm issue does seem to differ then. In our case it actually doesn't IPC timeout but timeout in the actual processing flow. Guess I'll have to look more into the issue on i.MX separately.

Reply:
@paulstelian97 If corresponding SSP is not enabled in BIOS, aplay/arecord will fail either. I am not familiar with ARM, but you should have the same setting in some place.

Reply:
Yeah again, that's not the nature of the issue (first aplay succeeds, the second one times out). I'll check it out because we suspect the DMA driver isn't perfect.